app/Views/giohang/index.php

```diff
@@ -0,0 +1,105 @@
+                        <th class="px-6 py-4 text-right text-xs font-bold text-muted-foreground uppercase">Thành tiền</th>
+                        <th class="px-4 py-4"></th>
+                    </tr>
+                </thead>
+                <tbody class="divide-y divide-border bg-background">
+                    <?php
+                    $tong = 0;
+                    foreach ($cartItems as $index => $item):
+                        $bienThe = $item['bienThe'];
+                        $qty = $item['qty'];
+                        $sanPham = $bienThe->getSanPham();
+                        $thanhTien = $bienThe->getGia() * $qty;
+                        $tong += $thanhTien;
+                        // Lấy ảnh đại diện biến thể (nếu có)
+                        $hinhAnhs = $bienThe->getHinhAnhs();
+                        $hinhAnhObj = is_array($hinhAnhs) ? ($hinhAnhs[0] ?? null) : ($hinhAnhs->first() ?: null);
+                        $image = $hinhAnhObj && method_exists($hinhAnhObj, 'getDuongDan')
+                            ? '/images/' . ltrim($hinhAnhObj->getDuongDan(), '/')
+                            : null;
+                    ?>
+                        <tr class="hover:bg-yellow-50 transition-colors">
+                            <td class="px-4 py-4">
+                                <?php if (!empty($image)): ?>
+                                    <img src="<?= htmlspecialchars($image) ?>" alt="<?= htmlspecialchars($bienThe->getTenDayDu()) ?>"
+                                        class="w-12 h-12 object-cover rounded-xl bg-slate-200 border border-border shadow" />
+                                <?php else: ?>
+                                    <span class="inline-flex items-center justify-center bg-muted rounded-full w-12 h-12">
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="none" stroke="currentColor" stroke-width="2"
+                                            stroke-linecap="round" stroke-linejoin="round">
+                                            <rect width="18" height="18" x="3" y="3" rx="2" />
+                                            <path d="m9 9 6 6M15 9l-6 6" />
+                                        </svg>
+                                    </span>
+                                <?php endif; ?>
+                            </td>
+                            <td class="px-6 py-4 text-base font-medium text-foreground">
+                                <?= htmlspecialchars($bienThe->getTenDayDu()) ?>
+                                <div class="text-xs text-muted-foreground"><?= htmlspecialchars($sanPham->getThuongHieu()->getTen() ?? '') ?></div>
+                            </td>
+                            <td class="px-6 py-4 text-center">
+                                <input type="number" name="quantities[<?= $index ?>]" value="<?= $qty ?>" min="1"
```

Comment:
Để nó tự submit form khi thay đổi số lượng
```suggestion
                                <input type="number" name="quantities[<?= $index ?>]" value="<?= $qty ?>" min="0" onchange="this.form.submit()"
```